tests/app/celery/v3/test_notification_tasks.py

```diff
@@ -267,6 +273,13 @@ def test_v3_process_notification_valid_sms_with_invalid_sender_id(
     v3_process_notification(request_data, sample_service.id, None, KEY_TYPE_TEST)
     v3_send_sms_notification_mock.assert_not_called()
 
+    query = select(NotificationFailures).where(NotificationFailures.notification_id == request_data["id"])
+    notification_failure = notify_db_session.session.execute(query).one()[0]
+    body = notification_failure.body
+
+    assert body.get('status') == NOTIFICATION_PERMANENT_FAILURE
+    assert body.get('status_reason') == "SMS sender None does not exist."
```

Comment:
please change to double-quotes to be consistent

Reply:
done